ports/serf/portfile.cmake

```
@@ -0,0 +1,33 @@
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://www.apache.org/dist/serf/serf-1.3.10.zip"
+    FILENAME "serf-1.3.10.zip"
+    SHA512 82e1c7342b0fa102c0e853989da0f6b590584e5a1d7737f891edd1d49b2a3ec271fd71f2642813455f73230c57230aebdc3a83808335dd53c5ce9fdab8506e2f
+)
+
+vcpkg_extract_source_archive(
+    SOURCE_PATH
+    ARCHIVE "${ARCHIVE}"
+    PATCHES
+        "serf-msbuild.patch"
+)
+
+vcpkg_install_msbuild(
+    SOURCE_PATH "${SOURCE_PATH}"
+    PROJECT_SUBPATH "serf.sln"
+    TARGET "serf"
+    PLATFORM ${VCPKG_TARGET_ARCHITECTURE}
+    OPTIONS
+        /p:CURRENT_INSTALLED_DIR=${CURRENT_INSTALLED_DIR}
+        /p:VCPKG_CRT_LINKAGE=${VCPKG_CRT_LINKAGE}
+)
+
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
+
+# Header files
+file(GLOB_RECURSE INCLUDES "${SOURCE_PATH}/*.h")
+file(COPY ${INCLUDES} DESTINATION "${CURRENT_PACKAGES_DIR}/include")

```

```suggestion
file(COPY ${INCLUDES} DESTINATION "${CURRENT_PACKAGES_DIR}/include/serf")
```

Place it under the `serf/` folder to avoid confusion with header files of other ports.

---

Thanks. I was confused because some libraries copy their include files to the `include` directory.